engine/kafka/src/main/scala/pl/touk/nussknacker/engine/kafka/KafkaRecordHelper.scala

```diff
@@ -0,0 +1,30 @@
+package pl.touk.nussknacker.engine.kafka
+
+import java.nio.charset.{Charset, StandardCharsets}
+
+import org.apache.kafka.common.header.Headers
+import org.apache.kafka.common.header.internals.RecordHeaders
+
+import scala.collection.JavaConverters._
+
+object KafkaRecordHelper {
```

Comment:
I wonder if it would be a good idea to have this code in companion object for `DeserializedConsumerRecord`. It looks like both things are related with transformation between kafka classes and our classes for some easier usage from scala or serialization or maybe I'm wrong?

Reply:
Changed to `ConsumerRecordUtils` to avoid name collision with `ConsumerRecord` from kafka.